.github/workflows/pre-release-check.yml

```diff
@@ -20,7 +20,7 @@ jobs:
         uses: actions/setup-java@v3
         with:
           distribution: temurin
-          java-version: 11
+          java-version: 17
```

Comment:
don't we want to run it in both 11 and 17?

Reply:
Ideally yes, but not sure it's worth the extra Dataflow cost... imo since our release config still targets Java 8 I think we're pretty well-covered as far as writing compatible code. wdyt?